sdk/loadtestservice/Azure.ResourceManager.LoadTestService/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("LoadTestService")]
```

Comment:
here is `Microsoft.LoadTestService` according to the swagger.

Reply:
The assembly info file has not been manually touched. I am not sure why auto-rest had not populated the naming as per the swagger. Please let us know if we can manually edit properties of this file, or should it be done as a part of code-gen.

Reply:
You can manually change this file. This file is generated by our dotnet template and the value depends on the argument that you pass, we do nothing to this file in code-gen.

Reply:
updated the namespace